src/main/resources/messages.properties

```
@@ -1039,3 +1039,12 @@ provider-responses-submit-start.expired.button=Return to home
 provider-responses-submit-start.responded.header=Hi, {0}. A response has already been recorded for this application.
 provider-responses-submit-start.responded.notice=The family's application has been sent to the CCR&R for processing.
 provider-responses-submit-start.responded.button=Return to home
+
+#contact-provider-message
+contact-provider-message.title=Contact Provider Message
+contact-provider-message.header=Message your child care provider
+contact-provider-message.below-we-prepared=Below, we've prepared a message that can be copied directly into an email or text to Sunnyside Daycare.
```

I know this is the POC so this is not a blocking comment. I'd recommend updating these messages to be more flexible.
```suggestion
contact-provider-message.below-we-prepared=Below, we've prepared a message that can be copied directly into an email or text to {0}.
```

---

In the future iteration.